### ISSUE TITLE
[Forge 1.4.7] Download link are dead for Forge.

### ISSUE THREAD

**Issue by Ertuit** [MultiMC/MultiMC5#3763]
### Operating System

Windows

### Description of bug

Hello,
The downloading of fml libraries is not possible because the download link are dead.

The log of the instance : (yes, I'm french :p )
```
MultiMC version: 0.6.12-1530


Minecraft folder is:
C:/Program Files/MultiMC/instances/1.4.7/.minecraft


Java path is:
C:/Program Files (x86)/Common Files/Oracle/Java/javapath/javaw.exe


Checking Java version...
Java is version 1.8.0_291, using 64-bit architecture, from Oracle Corporation.


Échec de la mise à jour de l'instance. Raison : Échec du téléchargement des fichiers suivants :
https://files.minecraftforge.net/fmllibs/argo-2.25.jar
https://files.minecraftforge.net/fmllibs/asm-all-4.0.jar
https://files.minecraftforge.net/fmllibs/bcprov-jdk15on-147.jar
https://files.minecraftforge.net/fmllibs/guava-12.0.1.jar

Raison : Échec du traitement de la tâche "FML libraries" :
https://files.minecraftforge.net/fmllibs/argo-2.25.jar
https://files.minecraftforge.net/fmllibs/asm-all-4.0.jar
https://files.minecraftforge.net/fmllibs/bcprov-jdk15on-147.jar
https://files.minecraftforge.net/fmllibs/guava-12.0.1.jar
Veuillez réessayer.

```

I think you must replace theses dead link with the wayback machine one. Who will probably never died.
Exemple for the guava-12.0.1.jar file :
https://web.archive.org/web/*/https://files.minecraftforge.net/fmllibs/guava-12.0.1.jar

Thank you, MultiMC is a amazing sofware.
Ertuit

### Steps to reproduce

Create a 1.4.7 Minecraft Instance.
Try to install Forge via the auto-button.

### Suspected cause

The download link are dead.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by jamierocks** [comment]
This has actually been resolved by https://github.com/MultiMC/MultiMC5/commit/a09d03d71d9a7fd89353a5bcec62caaa24d99230 :)

Having said that a release ought to be made, you could use the develop update channel in the meantime.

**Comment by Ertuit** [comment]
Oh, I'm sorry, I've not read the pull request section.
Thank you for your fast answer :)

**Comment by jamierocks** [comment]
No problem buddy, happy playing!

**Comment by DURMUSGUVEN2** [comment]
> This has actually been resolved by [a09d03d](https://github.com/MultiMC/Launcher/commit/a09d03d71d9a7fd89353a5bcec62caaa24d99230) :)
> 
> Having said that a release ought to be made, you could use the develop update channel in the meantime.

how can i do this i couldn't understand
